### PR TITLE
CHK-464: Update enqueue to aggregate orderForm messages

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?

Aggregates the orderForm messages of intermediate tasks, so we call show the user all the messages related to the items they have changes. E.g.: if a user updates an item quantity to above the maximum quantity of SKUs the store has configured, and quickly adds another product to the cart which changes the selected shipping option, they would only see the messages regarding this last addition, and would not be aware that they've updated the quantity of the first item to above the store's limit.

#### How to test it?

[Workspace](https://toasts--checkoutio.myvtex.com/).

Add the first product on the shelf and update its quantity to 51, then add the second product on the shelf. You should see both messages like I described above.

#### Describe alternatives you've considered, if any.

This logic shouldn't be here. We need to make an effort and move this and the `if (queueStatusRef.current === QueueStatus.FULFILLED) { setOrderForm(...) }` logic into `order-manager/OrderQueue` or `order-manager/OrderForm` where we would benefit from this in every other orchestrator without any code changes.

#### Related to / Depends on

N/A

### CODE REVIEW

**Comment by kaisermann** [comment]
I'm not managing to see the second toast by following the mentioned steps 🤔

File: react/.eslintrc [hunk 1]
```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "vtex-react"
+  "extends": "vtex-react",
+  "parserOptions": {
+    "project": "./react/tsconfig.json"
```
Comment (klzns) [inline react/.eslintrc:4]: 🤔  what does it do?

File: react/typings/vtex.render-runtime.d.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* Typings for `render-runtime` */
 declare module 'vtex.render-runtime' {
-  import { Component, ComponentType, ReactElement, ReactType } from 'react'
+  import { ComponentType, ReactElement, ElementType } from 'react'
```
Comment (klzns) [inline react/typings/vtex.render-runtime.d.ts:3]: You can use the render-runtime exported types instead of defining them

**Comment by lucasecdb** [comment]
> I'm not managing to see the second toast by following the mentioned steps 🤔

@kaisermann that's strange... I've attached a gif below showing what should happen, did you do the same as in the gif?

gif

File: react/.eslintrc (hunk 1 above)
Comment (lucasecdb) [reply react/.eslintrc:4]: it's necessary for some eslint rules, but I don't know why it didn't pickup this definition from [eslint-config-vtex](https://github.com/vtex/typescript/blob/master/packages/eslint-config-vtex/rules/typescript.js#L21-L29), so I had to manually add it.

File: react/typings/vtex.render-runtime.d.ts (hunk 2 above)
Comment (lucasecdb) [reply react/typings/vtex.render-runtime.d.ts]: removed the file, this app doesn't use `render-runtime` and the typings inside this file weren't correct anyway

File: react/.eslintrc (hunk 1 above)
Comment (lucasecdb) [reply react/.eslintrc]: nevermind, all I needed to do was run `yarn upgrade` 😄

**Review by kaisermann — APPROVED** [review]

**Review by klzns — APPROVED** [review]

**Review by darlenedms — APPROVED** [review]

**Pull request merged**